pages/cart.js

```diff
@@ -0,0 +1,251 @@
+import { useContext, useEffect, useState } from "react";
+import styled from "styled-components";
+import { toast } from "react-toastify";
+import { FaTimes } from "react-icons/fa";
+
+import Header from "../components/Header";
+import Footer from "../components/Footer";
+import Divider from "../styles/UIKit/Divider";
+import UIBUtton from "../styles/UIKit/Button";
+import useApi from "../utils/useApi";
+import CartConstants from "../config/CartConfig";
+import AuthContext from "../contexts/AuthContext";
+
+const Cart = () => {
+  const api = useApi();
+  const [cartItems, setCartItems] = useState([]);
+  const [cartData, setCartData] = useState({});
+  const [cartTotal, setCartTotal] = useState(0);
```

Comment:
Why create separate states when everything is already in the cart object, use the cart object directly from context.

Reply:
this has been taken care